docker_services_cli/docker-services.yml

```diff
@@ -32,8 +32,8 @@ services:
       - "POSTGRES_DB=${POSTGRESQL_DB}"
     ports:
       - "5432:5432"
-  es:
```

Comment:
comment: we were using `es` to be consistent with invenio (e.g. with cookiecutter-invenio-instance and co). That said, I also prefer `elasticsearch`.

Reply:
To put it simply:
- `es` is part of `postgresql`: having it named `es` is a major blocker when implementing `override_default_env`. 
- `es` doesn't follow the naming scheme of the rest of services in the docker compose file  or anywhere in the repo. All services are called full name, `es` is short.

Reply:
ohh damn! didn't notice it being part of the `postgrESql` :O